src/app/auth/userlogin/userlogin.component.html

```diff
@@ -1,41 +1,74 @@
-<main id="content" class="usa-section bypass-block-target">
-  <div class="grid-container">
-    <div class="usa-alert usa-alert--error" id="login-error-wrapper" hidden>
-      <div class="usa-alert__body">
-        <div class="usa-alert__text" id="login-error"></div>
-      </div>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="https://designsystem.digital.gov/stylesheets/uswds.min.css">
+</head>
+<body>
+  <div class="button-container">
+    <div class="login-container">
+      <span class="login-text">Sign in with:</span>
+      <a class="login-image" href="javascript:void(0);" onclick="(function() {
+        var client_host = window.location.hostname;
+        var server_host = '';
+        if (client_host === 'localhost') { server_host = 'localhost:3000'; }
+        if (client_host === 'srt-client.app.cloud.gov' || client_host === 'srt.app.cloud.gov') { server_host = 'srt-server.app.cloud.gov'; }
+        if (client_host === 'srt-client-dev.app.cloud.gov') { server_host = 'srt-server-dev.app.cloud.gov'; }
+        if (client_host === 'srt-client-staging.app.cloud.gov') { server_host = 'srt-server-staging.app.cloud.gov'; }
+        if (server_host === '') { server_host = 'localhost:3000'; }
+        window.location = 'http://' + server_host + '/api/login';
+      }())">
+        <img src="assets/login-gov-logo.svg" alt="Login.gov">
+      </a>
     </div>
 
-    <div class="display-flex flex-column flex-align-center">
-      <button *ngIf="is_dev" 
-              class="usa-button usa-button--outline margin-2"
-              id="login-dev"
-              aria-label="Login as Dev"
-              onClick=" (function() {
-                        var client_host = window.location.hostname;
-                        var server_host = '';
-                        if (client_host === 'localhost') { server_host = 'localhost:3000'; }
-                        if (server_host === '') { server_host = 'localhost:3000'; }
-                        window.location = 'http://' + server_host + '/api/casLogin';
-                      }()) ">
-        Login as DEV
-      </button>
+    <!-- Spacer div to add padding between the buttons -->
+    <div class="spacer"></div>
```

Comment:
You need to include ```*ngIf="is_dev"``` in this div. If not the login as dev button will show up in the other environments.